spec/features/notes_on_merge_requests_spec.rb

```diff
@@ -50,18 +50,18 @@
       end
 
       it 'should be added and form reset' do
-        should have_content("This is awsome!")
+        is_expected.to have_content("This is awsome!")
         within('.js-main-target-form') do
           expect(page).to have_no_field('note[note]', with: 'This is awesome!')
           expect(page).to have_css('.js-md-preview', visible: :hidden)
         end
-        within(".js-main-target-form") { should have_css(".js-note-text", visible: true) }
+        within(".js-main-target-form") { is_expected.to have_css(".js-note-text", visible: true) }
```

Comment:
Line is too long. [98/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.